README.md

```diff
@@ -1,21 +1,50 @@
+**<img src="https://raw.githubusercontent.com/SergelsOrg/csv2tex/main/src/main/resources/org/example/csv2tex/ui/openmoji_flag-germany_1f1e9-1f1ea.png" alt="Flag DE" width="20"/>German version / Deutsche Version:** [README_DE.md](README_DE.md)
```

Comment:
I would have preferred a relative path to the image in this repository, but somehow it did not work for me :shrug: